.github/workflows/packages.yml

```
@@ -0,0 +1,55 @@
+# Useful resource: https://github.com/actions/starter-workflows
+
+name: Check Packages
```

```suggestion
name: SDK
```

and rename to sdk.yml